src/themes/dark.scss

```diff
@@ -185,4 +185,11 @@
   --wxc-tooltip-background: #{$wxc-black-alpha-100};
   --wxc-tooltip-border: 1px solid #{$wxc-white-alpha-20};
   --wxc-tooltip-border-color: #{$wxc-white-alpha-20};
+
+  --wxc-default-background-color: #{$wxc-gray-95};
```

Comment:
these are related to adaptive cards, let's prefix them as such:

```css
--wxc-adaptive-cards--background-color--accent: #{$wxc-blue-80};
--wxc-adaptive-cards--background-color--emphasis: #{$wxc-gray-80};
```

and let's use alpha order